data/presets/leisure/bathing_place.json

```diff
@@ -0,0 +1,45 @@
+{
+    "aliases": [
+        "Bathing spot",
+        "Dip spot",
+        "Dipping spot",
+        "Swim spot",
+        "Swimming spot",
+        "Wild swimming spot"
```

Comment:
Could you double check the convention of `aliases` in other presets. I think they are either all lower case or "Camel Case".

Reply:
Camel Case! Will update